docs/monitors/collectd-signalfx-metadata.md

```diff
@@ -28,10 +28,11 @@ Monitor Type: `collectd/signalfx-metadata`
 | `etcPath` | no | `string` | The path to the main host config dir. Useful to override in containerized environments. (**default:** `/etc`) |
 | `persistencePath` | no | `string` | A directory where the metadata plugin can persist the history of successful host metadata syncs so that host metadata is not sent redundantly. (**default:** `/var/run/signalfx-agent`) |
 | `omitProcessInfo` | no | `bool` | If true, process "top" information will not be sent.  This can be useful if you have an extremely high number of processes and performance of the plugin is poor. (**default:** `false`) |
-| `dogStatsDPort` | no | `unsigned integer` | Set this to a non-zero value to enable the DogStatsD listener as part of this monitor.  The listener will accept metrics on the DogStatsD format, and sends them as SignalFx datapoints to our backend.  **Note: The listener emits directly to SignalFx and will not be subject to filters configured with the SignalFx Smart Agent.  Internal stats about the SignalFx Smart Agent will not reflect datapoints set through the DogStatsD listener** (**default:** `0`) |
+| `dogStatsDPort` | no | `unsigned integer` | Set this to a non-zero value to enable the DogStatsD listener as part of this monitor.  The listener will accept metrics on the DogStatsD format, and sends them as SignalFx datapoints to our backend.  **Note: The listener emits directly to SignalFx and will not be subject to filters configured with the SignalFx Smart Agent.  Internal stats about the SignalFx Smart Agent will not reflect datapoints set through the DogStatsD listener** |
```

Comment:
This will technically allow a zero value as well right?  Not sure there is any practical value in mentioning it to users since they probably want a fixed port though.